lib/modules/module.rb

```diff
@@ -50,5 +50,11 @@ def prescribeMedication(med, reason, time, entity)
         entity[:medications][med][1] << reason
       end
     end
+
+    def stopMedication(med, reason, time, entity)
+      return if entity[:medications][med].nil?
+      entity[:medications][med][1].delete(reason) if entity[:medications][med][1].include?(reason)
+      entity[:medications].delete(med) if entity[:medications][med].empty?
```

Comment:
`entity[:medications][med]` will never be empty because it will always contain a time, right?